scripts/python/lib/argparse_gen.py

```diff
@@ -292,14 +298,15 @@ def _check_deploy(args, subparser):
             args.install_client_os == ABSENT and
             args.ssh_keyscan == ABSENT and
             args.gather_mac_addr == ABSENT and
+            args.lookup_interface_names == ABSENT and
             args.config_client_os == ABSENT and
             args.all == ABSENT):
         subparser.error(
             'one of the arguments --create-inventory --install-cobbler'
             ' --inv-add-ports-pxe --inv-add-ports-ipmi --download-os-images'
             ' --add-cobbler-distros --add-cobbler-systems --install-client-os'
-            ' --ssh-keyscan --gather-mac-addr --config-client-os -a/--all is'
-            ' required')
+            ' --ssh-keyscan --gather-mac-addr --lookup-interface-names '
```

Comment:
I question whether lookup interface names needs to be callable through the CLI, but if it is to be callable, we should include more content under help.

Reply:
I don't mind either way. I updated the help message in eb4f677